torchrl/modules/models/multiagent.py

```diff
@@ -229,6 +229,141 @@ def forward(self, *inputs: Tuple[torch.Tensor]) -> torch.Tensor:
 
         return output
 
+class MultiAgentConvNet(nn.Module):
```

Comment:
we need to add it to the docs.
it will have to be added in one of the .rst files in the doc folder. search fro the mlp to see where